applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -336,27 +340,30 @@ int cloud_encode_data(const struct cloud_channel_data *channel,
 	return 0;
 }
 
-int cloud_encode_digital_twin_data(const struct cloud_channel_data *channel,
-				   struct cloud_msg *output)
+int cloud_encode_device_status_data(struct modem_param_info *modem_param,
```

Comment:
Please either move all parameters below the opening bracket, or align.

Reply:
OK, I will do this tomorrow.  Do not merge until then.

Reply:
Done, and also fixed the gitlint problems.